### PR TITLE
wording from 3.4

### PR DESCRIPTION
**Pull request by dietertaotesting**

### CODE REVIEW

File: User Guide 3.3/Deliveries/create-a-new-delivery.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ A properties panel will come up.
 
 **Maximum Executions:** If left empty, test-takers may take the delivered test an unlimited number of times. Setting this number to any non-zero integer will limit the test-takers to that number of attempts.
```
Comment (dietertaotesting) [inline User Guide 3.3/Deliveries/create-a-new-delivery.md:34]: What about 'If this remains empty'?

File: User Guide 3.3/Introduction/customizing-look-and-feel.md [hunk 2]
```diff
@@ -37,6 +41,6 @@ Now add the information *Operated by* to the page footer. You can add your organ
 
 If you are happy with the information you have entered, click on  *Apply changes* to save it. If not, click on *Discard changes*.
 
-Your institution or company may also want to provide direct access to the TAO environment from its own website. As a subscriber to the Premium Edition of TAO, your organization can set up a shortcut to the TAO environment via a URL which you own, to refer users directly to TAO. 
+Your institution or company may also want to provide direct access to the TAO environment from its own website. As a subscriber to the Ignite Edition (or higher) of TAO, your organization can set up a shortcut to the TAO environment via a URL which you own, to refer users directly to TAO.
```
Comment (cathpease) [inline User Guide 3.3/Introduction/customizing-look-and-feel.md:44]: should be: Premium/Ignite

File: User Guide 3.3/Items/items-an-overview.md [hunk 3]
```diff
@@ -1,25 +1,19 @@
...
+Items first need to be created before they are populated with the desired [Interactions](../appendix/glossary.md#interaction) and any further material required so they can be used in assessments. See [Creating a new Item](../items/creating-a-new-item.md) for information on how to do this.
 
 **2.** What to include in your item
 
-Pictures and media such as audio or video files can be included in your Item. For more information on how to do this, see [Adding Media](../items/adding-media.md), for adding media to your items, and [Media Interactions](../interactions/media-interaction.md) for adding audio and video files.
+Pictures and media such as audio or video files can be included in your Item. For more information on how to add media to your items, see [Using Assets in Items](../items/using-assets-in-items.md). To create media interactions see the section on [Media Interactions](../interactions/media-interaction.md).
```
Comment (cathpease) [inline User Guide 3.3/Items/items-an-overview.md:14]: change asset file reference to 'Adding Media'

File: User Guide 3.3/Items/items-an-overview.md [hunk 4]
```diff
@@ -1,25 +1,19 @@
...
 
-For managing resources which are included in your items (which includes all of the resources mentioned above), see the [Media Manager](../items/media-manager.md) section.
+For managing resources which are included in your items (which includes all of the resources mentioned above), see the [Asset Manager](../items/asset-manager.md) section.
```
Comment (cathpease) [inline User Guide 3.3/Items/items-an-overview.md:16]: as above

File: User Guide 3.3/Taking a Test/text-to-speech.md [hunk 5]
```diff
@@ -9,24 +9,21 @@ authors:
 
 >[Test-takers](../appendix/glossary.md#test-taker) can elect to use the text-to-speech functionality for a [Test](../appendix/glossary.md#test).
 
-The text-to-speech functionality allows the test-taker to hear the test questions, or parts of them, read aloud.
+The text-to-speech functionality allows you to hear the test questions, or parts of them, read aloud.
 
-*Note: This functionality is only available in the Premium Edition of TAO.*
+*Note: This functionality is only available in the Ignite Edition (or higher) of TAO.*
```
Comment (cathpease) [inline User Guide 3.3/Taking a Test/text-to-speech.md:14]: Premium/Ignite

File: User Guide 3.3/Tests/test-settings.md [hunk 6]
```diff
@@ -24,69 +18,81 @@ See the section on [Scoring Rules](../tests/test-scoring-and-outcome-declaration
...
 
 The general properties include entries for: *Identifier*, *Reference*, and *Categories*. It also includes check boxes to indicate if the item is *Required* and if it is *Fixed*.
 
 ![Settings for Test Items](../resources/backend/tests/authoring/settings/item/properties.png)
 
 The *Identifier* and *References* boxes generally do not require modification. *Categories* act as tag references which may be displayed to the test-taker. If the *Required* box is checked, the item will appear in the test, even if less than the total number of items appears in a given test (in a section where selection is enabled). If the *Fixed* box is checked, the item will appear in a particular order, even if the section ordering calls for shuffling. Categories are not displayed to the test-taker; typically they are used to calculate aggregate scores (e.g. on sub-domains). For more information, see the section on [Scoring Rules](../tests/test-scoring-and-outcome-declarations.md).
 
+For *Test Navigation*, *Navigation Warnings* and *Test-taker Tools*, see thes description in the section level properties above.
```
Comment (cathpease) [inline User Guide 3.3/Tests/test-settings.md:94]: see the description

File: User Guide 3.3/Introduction/customizing-look-and-feel.md (hunk 2 above)
Comment (cathpease) [reply User Guide 3.3/Introduction/customizing-look-and-feel.md]: changed

File: User Guide 3.3/Items/items-an-overview.md (hunk 3 above)
Comment (cathpease) [reply User Guide 3.3/Items/items-an-overview.md:14]: changed to adding media

File: User Guide 3.3/Items/items-an-overview.md (hunk 4 above)
Comment (cathpease) [reply User Guide 3.3/Items/items-an-overview.md:16]: name and path changed (../managing-media/media-manager.md)

File: User Guide 3.3/Taking a Test/text-to-speech.md (hunk 5 above)
Comment (cathpease) [reply User Guide 3.3/Taking a Test/text-to-speech.md]: 'premium' now in

File: User Guide 3.3/Tests/test-settings.md (hunk 6 above)
Comment (cathpease) [reply User Guide 3.3/Tests/test-settings.md]: corrected

**Pull request merged**